### PR TITLE
[DOCS-6401] Add prereq for using Search Services with Outlook Integration

### PR DESCRIPTION
**Pull request by anxumalo**
Adds a prerequisite for using Alfresco Search Services or Alfresco Search and Insight Engine 2.0 or above with Alfresco Outlook Integration 2.8.1 and above.

**Note:** This PR replaces draft PR #365.

### CODE REVIEW

**Comment by anxumalo** [comment]
Hi @mhofmann-itnovum, this update is ready for an initial review. I've now closed your original PR as it's covered by this one. Please see my email for additional discussion points to consider for this release. Many thanks

File: microsoft-outlook/latest/install/index.md [hunk 1]
```diff
@@ -58,7 +58,22 @@ You can use one of the following Outlook releases:
 
 ### Alfresco requirements
 
-* Alfresco Content Services 6.2.2 or later. See [Supported Platforms]({% link microsoft-outlook/latest/support/index.md %}) for more information.
+* Alfresco Content Services 6.2.2 or later.
+
+#### Alfresco Search and Insight Engine 2.0 and above
```
Comment (mhofmann-itnovum) [inline microsoft-outlook/latest/install/index.md:63]: Change to Alfresco Search Services 2.0 and above

File: microsoft-outlook/latest/install/index.md [hunk 2]
```diff
@@ -58,7 +58,22 @@ You can use one of the following Outlook releases:
...
+
+If you're using Alfresco Search and Insight Engine 2.0 and above in combination with Outlook Integration 2.8.1 and above, a property needs to be added to the `shared.properties` file for SOLR. See the [Alfresco indexing recommendations]({% link insight-engine/latest/config/indexing.md %}#cross-locale) to locate this file.
```
Comment (mhofmann-itnovum) [inline microsoft-outlook/latest/install/index.md:65]: replace "Alfresco Search and Insight Engine 2.0 and above" with
"Alfresco Search Services 2.0 and above"

**Review by mhofmann-itnovum — APPROVED** [review]
OK for me after the suggested text change.

**Comment by anxumalo** [comment]
@mhofmann-itnovum, @pnirwal - these changes are now ready for review. Thanks

**Review by mhofmann-itnovum — APPROVED** [review]

**Pull request merged**